### ISSUE TITLE
[bug]: laggy text input

### ISSUE THREAD

**Issue by cdaringe** [hoppscotch/hoppscotch#3762]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

When I type, characters are extremely delayed entering the screen.

### Steps to reproduce

1. download the macos arm build
2. create a new collection. i only have four POST routes
3. execute some requests
4. type. observe chronic slowness

### Environment

Production

### Version

Local

**Comment by CheaterTim** [comment]
I am encountering the same issue on linux with the deb, I have found that mostly writing into the request body seems to cause issues (syntax highlighting refreshing?), but I haven't tested it extensively yet.

I am not qualified to debug issues like this, but I did my best in the dev tools, maybe it'll be useful to someone.
![image](https://github.com/hoppscotch/hoppscotch/assets/85891155/5a68782f-aa80-4cdb-bf13-26ce6afe3d95)

**Comment by HeIIow2** [comment]
This is defently a huge issue.

Writing in the request body or or getting the formatted response nearly brings the whole program to a halt.
(every action takes about a second)

I am also using the `.deb` file. My specs are:
```  
OS: Pop!_OS 22.04 LTS x86_64
Host: MacBookPro16,2 1.0
Kernel: 6.6.16-t2-jammy
DE: GNOME 42.5
WM: Mutter
CPU: Intel i5-1038NG7 (8) @ 3.800GHz
GPU: Intel Device 8a53
```

Disabling line wrap seems to help a bit, but not that much.

**Comment by HeIIow2** [comment]
Here is the code that adds the onchange listener to the body, regardless of formatting.

https://github.com/hoppscotch/hoppscotch/blob/00862eb192f4df30f564d408b48ba9843d48b0c8/packages/hoppscotch-common/src/composables/codemirror.ts#L289

**Comment by HeIIow2** [comment]
@cdaringe The bug was fixed for me using the newest release of the desktop app.
Could you test it and close the issue if resolved?

**Comment by liyasthomas** [comment]
Thanks for your patience and valuable feedback. This issue has been resolved in the latest release via #3848 on [v2023.12.6](https://github.com/hoppscotch/hoppscotch/releases/tag/2023.12.6).

Closing this ticket as the fix is now available in the recent version. Feel free to reach out if you encounter any further concerns.
